Specification/1._Introduction.md

```diff
@@ -4,23 +4,19 @@ permalink: /Specification/1._Introduction/
 
 # Introduction
 
-This section covers basic concepts that govern development and maintenance of the specification.
+This section covers the basic concepts that govern development and maintenance of the specification.
 The actual specification is contained in the following sections.
 
 ## Core design goals
 
-The list of core design goals below helps to understand the basic concepts of UAVCAN and the motivation behind them.
+The list of core design goals below help explain the basic UAVCAN concepts and the motivation behind them.
```

Comment:
Perhaps my grammar is a bit off here, but in the next above "help" is referring to "list", which is a singular subject ("the list helps").